controllers/certificatepolicy_controller_test.go

```diff
@@ -31,80 +31,26 @@ import (
 	testclient "k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/kubernetes/scheme"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	policiesv1 "open-cluster-management.io/cert-policy-controller/api/v1"
 )
 
-func TestReconcile(t *testing.T) {
-	name := "foo"
-	namespace := "default"
-	instance := &policiesv1.CertificatePolicy{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "foo",
-			Namespace: "default",
-		},
-		Spec: policiesv1.CertificatePolicySpec{
```

Comment:
Are we losing some tests that validate the durations?

Reply:
It's a bit confusing but since the Reconcile method only added and removed entries in the map, this didn't actually evaluate the policy.

Reply:
It looks like the certificate expiration is covered in the e2e test.